nimble/host/src/ble_gap.c

```diff
@@ -3007,6 +3007,44 @@ ble_gap_ext_adv_remove(uint8_t instance)
     return 0;
 }
 
+int
+ble_gap_ext_adv_clear(void)
+{
+    int rc;
+    uint8_t instance;
+    uint16_t opcode;
+
+    ble_hs_lock();
+
+    for(instance = 0; instance < BLE_ADV_INSTANCES; instance++) {
+        /* If there is an active instance or periodic adv instance,
+         * Don't send the command
+         * */
+        if ((ble_gap_slave[instance].op == BLE_GAP_OP_S_ADV)
+#if MYNEWT_VAL(BLE_PERIODIC_ADV)
```

Comment:
I would just add separate `if ( ` for periodic check as it will be cleaner and not against wrapping long lines coding style (https://github.com/apache/mynewt-core/blob/master/CODING_STANDARDS.md)

Reply:
Modified.